Libraries/pbxbuild/Sources/Target/Environment.cpp

```diff
@@ -440,7 +440,10 @@ Create(Build::Environment const &buildEnvironment, Build::Context const &buildCo
     }
 
     /* Tool search directories. Use the toolchains just discovered. */
-    std::vector<std::string> executablePaths = sdk->executablePaths(toolchains);
+    std::vector<std::string> executablePaths;
+    if (auto manager = sdk->manager()) {
```

Comment:
Could you use `buildEnvironment.sdkManager()` here instead? It's always available and doesn't depend on the unfortunate nullable backpointers in the SDK objects.